charts/localstack/templates/deployment.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "localstack.fullname" . }}
-  namespace: {{ .Release.Namespace | quote }}
+  namespace: {{ include "localstack.namespace" . }}
```

Comment:
```suggestion
  namespace: {{ include "common.names.namespace" . | quote }}
```

Reply:
I was working on implementing this change when I noticed that the project uses localstack.name, localstack.fullname, and localstack.chart. We have two options: either we move everything under common.names.xxxx for consistency, or we maintain the current standard and continue using localstack.xxx. What do you think? For reference, other libraries like common have a separate file for names, such as _names.tpl.

Reply:
I think trying to consolidate everything by using the `common` chart utils functions is out of scope for this PR.
I also don't see a reason to create an unnecessary layer of indirection by creating a new `localstack.namespace` if it is just going to use `common.names.namespace`.
So in my opinion, we should just use `common.names.namespace` everywhere (which is a backwards compatible change), no other changes necessary.

Reply:
I've just pushed a change. Please review it and let me know if it meets the expected requirements